ops/lib/environment.js

```diff
@@ -1,9 +1,7 @@
 const yargs = require('yargs');
 
 var environment = {
-  production: !!(yargs.argv.production),
-  development: !(yargs.argv.production),
-  debug: !!(yargs.argv.debug),
+  production: !!(yargs.argv.production)
```

Comment:
Are the other environments not used anymore?

Reply:
Nope - this last one is just used in pkg-less that Jeffrey is working on replacing.